client/src/components/settings/appearance-management/element-preview.js

```diff
@@ -131,6 +130,8 @@ class ElementPreview extends React.Component {
   renderPreviews () {
     const {section} = this.props;
     switch (section) {
+      case sectionNames.backgroundImage:
+        return this.renderBackgroundImage();
```

Comment:
This component does not have `renderBackgroundImage` method

Reply:
fixed